### PR TITLE
Update tracker, gnome cases for x11regression

### PR DESCRIPTION
**Pull request by chuchingkai**
Testing Result：

http://147.2.212.239/tests/554

### CODE REVIEW

File: tests/x11regressions/tracker/tracker_by_command.pm [hunk 1]
```diff
@@ -17,7 +17,7 @@ use testapi;
 sub run() {
     my $self = shift;
     x11_start_program("xterm");
-    script_run "tracker-search newfile";
+    script_run "tracker search newfile";
```
Comment (coolo) [inline tests/x11regressions/tracker/tracker_by_command.pm]: will the new method also work in SP1 maintenance?

File: tests/x11regressions/tracker/tracker_searchall.pm [hunk 2]
```diff
@@ -18,15 +18,6 @@ sub run() {
     sleep 2;
     wait_idle;
     assert_screen 'tracker-needle-launched';
-    send_key "tab";
```
Comment (coolo) [inline tests/x11regressions/tracker/tracker_searchall.pm]: this now looks definitly like this will break SP1 maintenance

Comment (okurz) [inline tests/x11regressions/tracker/tracker_searchall.pm]: I guess we now reached the point where it makes sense to create maintenance branches for the tests and needles repo.

Comment (chuchingkai) [inline tests/x11regressions/tracker/tracker_searchall.pm]: @coolo I see, I will add 

    if (get_var("SP2ORLATER"))

to make it works on both SP1 and SP2

File: tests/x11regressions/gnomecase/gnome_default_applications.pm [hunk 3]
```diff
@@ -61,7 +61,7 @@ sub run() {
     send_key "ctrl-w";                                     #close fileroller
     wait_still_screen;
     assert_and_dclick "gnomecase-defaultapps-htmlfile";    #open html
-    assert_screen 'gnomecase-defaultapps-firefoxopen';
+    assert_screen 'gnomecase-defaultapps-firefoxopen', 60;
```
Comment (okurz) [inline tests/x11regressions/gnomecase/gnome_default_applications.pm]: will it work if you just delete the timeout and rely on the timeout? code would be cleaner.

**Comment by okurz** [comment]
please don't call a commit "fixup". This should be reserved for [autosquashing git commits](https://robots.thoughtbot.com/autosquashing-git-commits). Also see http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.

File: tests/x11regressions/gnomecase/gnome_default_applications.pm (hunk 3 above)
Comment (chuchingkai) [inline tests/x11regressions/gnomecase/gnome_default_applications.pm]: @okurz My local server has a low performance, and the default timeout (30 s) is not enough sometimes.

**Comment by chuchingkai** [comment]
@okurz I will correct the commit message, thanks.

File: tests/x11regressions/gnomecase/gnome_default_applications.pm (hunk 3 above)
Comment (okurz) [inline tests/x11regressions/gnomecase/gnome_default_applications.pm]: On Monday 30 May 2016 23:22:17 chuchingkai wrote:
> > -    assert_screen 'gnomecase-defaultapps-firefoxopen';
> > +    assert_screen 'gnomecase-defaultapps-firefoxopen', 60;
> 
> @okurz My local server has a low performance, and the default timeout (30 s)
> is not enough sometimes.

try setting setting the TIMEOUT_SCALE variable for your worker, e.g. 
TIMEOUT_SCALE=2
This variable is used for slow workers.

**Pull request merged**